backend/tests/test_beta.py

```diff
@@ -0,0 +1,135 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+- Fail setting own beta status because beta is disabled
+- Fail setting own beta status because self serve beta is disabled
+- Fail setting other people's beta status because not an admin
+- All of the above but successful
+- Enable and disable beta
+"""
```

Comment:
Put the docstrings on the individual tests itself. Otherwise it's hard to keep them up-to-date or identify which message belongs to which test.

Reply:
Oops that was meant for Copilot, but I can move them to the individual tests 😆